start-site/src/main/resources/application.yml

```diff
@@ -36,6 +36,8 @@ initializr:
             version: 2.1.10
           - compatibilityRange: "2.2.0.M1"
             version: 2.2.4
+          - compatibilityRange: "2.3.0.RELEASE"
```

Comment:
I don't see this version being built with Spring Boot `2.3.0.RELEASE`.  There is also an [open issue](https://github.com/microsoft/azure-spring-boot/issues/897) that leads me to believe this is unresolved.

Reply:
* I released version 2.3.0 by tag `v2.3.0`, it's built with Spring Boot `2.3.0.RELEASE`: https://github.com/microsoft/azure-spring-boot/blob/v2.3.0/azure-spring-boot-bom/pom.xml#L49
* Now it can be accessed in maven repository: https://repo.maven.apache.org/maven2/com/microsoft/azure/azure-spring-boot-bom/2.3.0/

Reply:
Did you try to release `2.3.0` twice or something? I can see a `2.3.0` release [on Maven Central](https://repo1.maven.org/maven2/com/microsoft/azure/azure-spring-boot-parent/2.3.0/azure-spring-boot-parent-2.3.0.pom) that has a reference to Spring Boot 2.2.

In any case, we don't need a release that builds on Spring Boot 2.3. What's important is that you have a CI job that builds the project and run the tests with Spring Boot 2.3. If you have one for Spring Boot 2.2 and Spring Boot 2.3 for the same version of the project, we're happy.

Reply:
> Did you try to release 2.3.0 twice or something? I can see a 2.3.0 release on Maven Central that has a reference to Spring Boot 2.2. The tag on Github for v2.3.0 refers to Spring Boot 2.3 so it looks like your tag does not match what was published on Maven Central.

Sorry, that's my mistake. Now I'm releasing 2.3.1, please wait.


> If you have one for Spring Boot 2.2 and Spring Boot 2.3 for the same version of the project, we're happy.

We will only depends on Spring Boot 2.3 on our 2.3.1 release.

Reply:
> Now I'm releasing 2.3.1, please wait.

Done:

https://repo1.maven.org/maven2/com/microsoft/azure/azure-spring-boot-parent/2.3.1/azure-spring-boot-parent-2.3.1.pom